script/build

```diff
@@ -5,18 +5,25 @@
 PLUGIN_NAME="${PLUGIN_NAME:?Must be set in .env}"
 PLUGIN_VERSION="${PLUGIN_VERSION:?Must be set in .env}"
 
+# This is necessary to get the project building on Go 1.6
+# Building the project will fail on Go 1.7+ as this env var will be ignored
+export GO15VENDOREXPERIMENT=0
```

Comment:
Sure. I'd like to leave the changes I currently have in, mostly because I've only got Go 1.6.2 (the environment variable is the only way it will build on the current version of Go) installed. I will raise a separate issue for supporting Go 1.6+ properly, though.